src/image.rs

```diff
@@ -28,6 +58,8 @@ pub enum LeafContent {
     CharacterDevice(u64),
     Fifo,
     Socket,
+    /// Contains the target of the link
+    /// The actual link path should be in TarEntry.path
```

Comment:
This comment seems a bit out of place (again: this code should have no knowledge about tar).